new-standalone-service.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+cleanup() {
+    # TODO(mohit): Maybe add a trap for SIGINT.
+    rm -rf template.zip
```

Comment:
If this is added as part of a trap, you'll probably need to give an absolute path to delete. The script may be in some random directory at the time of an error...